moveit_plugins/moveit_ros_control_interface/src/controller_manager_plugin.cpp

```diff
@@ -360,15 +377,17 @@ class MoveItControllerManager : public moveit_controller_manager::MoveItControll
       }
     }
 
+    // For each controller to activate, find conflicts between the interfaces required by that controller and the
+    // interfaces claimed by currently-active controllers.
     for (const std::string& it : activate)
     {
       ControllersMap::iterator c = managed_controllers_.find(it);
       if (c != managed_controllers_.end())
       {  // controller belongs to this manager
         request->start_controllers.push_back(c->second.name);
-        for (const auto& claimed_resource : c->second.claimed_interfaces)
+        for (const auto& required_resource : c->second.required_command_interfaces)
```

Comment:
What if Controller A requires the position command interface for Joint 1 and Controller B requires the velocity command interface for Joint1. Will that be seen as a conflict here? I think it should be.

(Refer to the [example here](https://github.com/ros-controls/ros2_control_demos/blob/37b3338b22ce99200791b191df647256d51f233e/ros2_control_demo_description/rrbot_description/ros2_control/rrbot_system_multi_interface.ros2_control.xacro#L30))

I think you will want to mark it as a conflict if both controllers have any command_interface for the same joint

Reply:
Ok, that's fine with me. Thanks for the comment.